hx_lti_initializer/views.py

```diff
@@ -369,7 +371,8 @@ def course_admin_hub(request):
             'debug': debug,
             'starter_object': starter_object,
             'starter_object_id': object_id,
-            'starter_collection_id': collection_id
+            'starter_collection_id': collection_id,
+            'utm_source': request.session.session_key if not request.session['is_staff'] else '',
```

Comment:
I noticed that in the `admin_hub.html` template, the **utm_source** parameter is missing from the URLs that students would click on to view the assignment objects.

Example:

```
<a href="{% url 'hx_lti_initializer:access_annotation_target' course_id=course.course_id assignment_id=assignment.assignment_id object_id=file.id %}">{{file.target_title}}</a>
```

Should be:


```
<a href="{% url 'hx_lti_initializer:access_annotation_target' course_id=course.course_id assignment_id=assignment.assignment_id object_id=file.id %}?utm_source={{utm_source}}">{{file.target_title}}</a>
```